### ISSUE TITLE
Example embedding generation script

### ISSUE THREAD

**Issue by JeffBezos64** [zouharvi/pwesuite#20]
Kia ora Vilém,

I have tried to implement a system to conduct word level embedding on arbitrary words for the RNNMetricLearner similar to how the sharma and parrish embeddings are done. However I have not been successful in writing this script and all of my embedding's are identical (not desired behaviour). You can see my [repo](https://github.com/JeffBezos64/zouhar_embedding) to verify my efforts. I have had significant difficulty decoupling your code to achieve this.

Can you please provide an example script for arbitrary words such that one could call `embed(word)` and receive an embedding? - I am looking at the metric learner and using "orth".

**Comment by zouharvi** [comment]
The instructions were only on HuggingFace, so I put them now also in the Readme. This is the code:
```
from models.metric_learning.model import RNNMetricLearner
from models.metric_learning.preprocessor import preprocess_dataset_foreign
from main.utils import load_multi_data
import torch
import tqdm
import math

data = load_multi_data(purpose_key="all")
data = preprocess_dataset_foreign(
  [
    {"token_ort": "Hello", "token_ipa": None},
    {"token_ort": "what", "token_ipa": None},
    {"token_ort": "is", "token_ipa": None},
    {"token_ort": "pwesuite", "token_ipa": None},
  ],
  features="token_ort"
)

model = RNNMetricLearner(
    dimension=300,
    feature_size=data[0][0].shape[1],
)
model.load_state_dict(torch.load("computed/models/rnn_metric_learning_token_ort_all.pt"))

# some cheap paralelization
BATCH_SIZE = 32
data_out = []
for i in tqdm.tqdm(range(math.ceil(len(data) / BATCH_SIZE))):
    batch = [f for f, _ in data[i * BATCH_SIZE:(i + 1) * BATCH_SIZE]]
    data_out += list(
        model.forward(batch).detach().cpu().numpy()
    )

assert len(data) == len(data_out)
assert all([len(x) == 300 for x in data_out])
```

**Comment by JeffBezos64** [comment]
Hi Vilém,

Thank you for your reply.

Yeah I based the code I was writing on the above sample and I moved all of the required functions into the class I was using. However in doing this I started receiving identical null vectors and couldn't make heads or tails of it.

Could you please modify the above to show the steps to create an embedding from an orth sample for the word "example"? This would illustrate using it in a context beyond PWESuite.

**Comment by zouharvi** [comment]
You just change one of the lines in the data to `"example"`, like so:
```
data = preprocess_dataset_foreign(
  [
    {"token_ort": "example", "token_ipa": None},
  ],
  features="token_ort"
)
```

**Comment by JeffBezos64** [comment]
Thank you! I was definitely doing it wrong.
